stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml

```diff
@@ -21,6 +21,9 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/header"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
```

Comment:
NIT: Those two margins can be replaced with `layout_marginHorizontal`

Reply:
Thanks